tests/reader.test.js

```diff
@@ -0,0 +1,222 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const { Reader, Book, Author, Genre } = require('../src/models');
```

Comment:
Book, Author and Genre are not used in this file so should be removed from the import